### PR TITLE
Create Sydney.json

### PR DESCRIPTION
**Pull request by thomaseizinger**

### CODE REVIEW

File: _data/events_gdcr2018/Sydney.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "title": "Sydney Global Day of Code Retreat 2018",
+  "url": "https://www.eventbrite.com.au/e/sydney-global-day-of-code-retreat-2018-tickets-51246992133",
+  "moderators": [
+    "Thomas Eizinger",
+    "Lloyd Fournier",
+    "Philipp Hoenisch",
+    "Franck Royer",
```
Comment (thodorisbais) [inline _data/events_gdcr2018/Sydney.json]: Please remove the `,`character here, so that the build can compile.

Then, we merge :-) .

Comment (thomaseizinger) [reply _data/events_gdcr2018/Sydney.json]: Ah, I guess too much Rust in last months :D

**Comment by thodorisbais** [comment]
Welcome @thomaseizinger , have a good one! :tada:

**Comment by thomaseizinger** [comment]
Thanks a lot! I am excited :)

**Pull request merged**